docs/userguide/networking/default_network/build-bridges.md

```diff
@@ -18,9 +18,10 @@ install Docker.
 create user-defined networks in addition to the default bridge network.
 
 You can set up your own bridge before starting Docker and use `-b BRIDGE` or
-`--bridge=BRIDGE` to tell Docker to use your bridge instead.  If you already
-have Docker up and running with its default `docker0` still configured, you will
-probably want to begin by stopping the service and removing the interface:
+`--bridge=BRIDGE` to tell Docker to use your bridge instead. If you already
+have Docker up and running with its default `docker0` still configured, you can
+directly create your own bridge and restart Docker with your bridge,
+or you will probably want to begin by stopping the service and removing the interface:
```

Comment:
I feel mixed about this sentence, I would have written either : 

- `If you already […] and restart Docker with your bridge, or want to begin by stopping the service […]`
- `If you already […] and restart Docker with your bridge; or you will probably want to begin […]`